groups/instance/instance.tf

```diff
@@ -0,0 +1,5 @@
+resource "aws_instance" "artifactory" {
+  ami           = var.ami
+  instance_type = var.default_instance_type
+  subnet_id = local.placement_vpc_pattern
+}
```

Comment:
There's a lot of parameters missing here.
You're not specifying the subnet IDs correctly, you've assigned the name of the VPC.
The `cloudinit_config` is not being associated
There's no specification for the required block devices
The `key_name` is missing
The security group is not being associated

Reply:
Block devices out of scope for this ticket.